regnety/utils/tfrecords_utils.py

```diff
@@ -0,0 +1,184 @@
+"""Script to make and save TFRecords from ImageNet files"""
+
+import tensorflow as tf
+import os
+import random
+import json
+import argparse
+import math
+from .image_utils import *
+
+
+def _bytes_feature(value):
+    """Returns a bytes_list from a string / byte."""
+    if isinstance(value, type(tf.constant(0))):
+        value = (
+            value
+        )  # BytesList won't unpack a string from an EagerTensor.
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=[value]))
+
+
+def _float_feature(value):
+    """Returns a float_list from a float / double."""
+    return tf.train.Feature(float_list=tf.train.FloatList(value=[value]))
+
+
+def _int64_feature(value):
+    """Returns an int64_list from a bool / enum / int / uint."""
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=[value]))
+
+
+# make_tfrecs -> for(make_single_tfrecord)-> make_dataset -> make_example
+def get_synset_labels(filepath):
+
+    with open(filepath, "r") as f:
+        raw_labels_dict = json.load(f)
+    labels_dict = dict()
+
+    for i in raw_labels_dict:
+        labels_dict[raw_labels_dict[i]["id"]] = (
+            int(i),
+            raw_labels_dict[i]["label"],
+        )
+    return labels_dict
+
+
+def get_files(data_dir, synset_filepath, shuffle = True):
+    all_images = tf.io.gfile.glob(os.path.join(data_dir, "*", "*.JPEG"))
+    all_synsets = [os.path.basename(os.path.dirname(f)) for f in all_images]
+
+    print(len(all_images))
```

Comment:
Should the print still be here? If so, consider adding some more context so it's clear what the number means.

Reply:
No, it was for debugging. I'll remove it.